docs/client/javascript-mono/react/_reactGatesAndConfigs.mdx

```diff
@@ -4,103 +4,92 @@ import GitHubEmbed from "@site/src/components/GitHubEmbed";
 
 You can get an instance of the StatsigClient to check gates, experiments, dynamic configs, layers, and log events.
 
-```typescript
-import { useStatsigClient } from "@statsig/react-bindings";
+{(() => {
+  switch(props.packageName) {
+    case 'react-native':
+      return (
+        <pre><code className="language-jsx">
+        {`import { useStatsigClient } from "@statsig/react-native-bindings";
+
+const { client } = useStatsigClient();`}
+        </code></pre>
+      );
+    case 'react-native-expo':
+      return (
+        <pre><code className="language-jsx">
+        {`import { useStatsigClient } from "@statsig/react-native-expo-bindings";
+
+const { client } = useStatsigClient();`}
+        </code></pre>
+      );
+    default:
+      return (
+        <pre><code className="language-jsx">
+        {`import { useStatsigClient } from "@statsig/react-bindings";
+
+const { client } = useStatsigClient();`}
+        </code></pre>
+      );
+  }
+})()}
+
+See the methods you can call on the client below.
 
-const { client } = useStatsigClient();
-```
 
 ### Checking a Feature Flag/Gate {#basics-check-gate}
 You can evaluate a gate by getting the client with the `useStatsigClient` hook,
 and then calling `checkGate`
 
 ```tsx
-// RootPage.js
-import { useStatsigClient } from "@statsig/react-bindings";
```

Comment:
I couldnt find a way to parameterize the code snippets with the package, so instead I pulled out the import example to the top, did the brute force switch statement, and then just showed the inline docs here.